app/assets/stylesheets/components/_event.sass

```diff
@@ -0,0 +1,30 @@
+$Event-color-background: Theme-color(background, default)
+$Event-color-borderTop: #FF623F  // FIXME Put this color on background color MAP on theme.sass file.
+$Event-height: 150px
+$Event-spacing-base: $Theme-spacing-base
+$Event-text-title-color: Theme-color(text, default)
+$Event-width: 280px
+
+.Event
+  background-color: $Event-color-background
+  border-radius: 3px
+  border-top: 5px solid $Event-color-borderTop
+  box-shadow: 0 2px 10px -3px gray
+  display: inline-block
+  height: $Event-height
+  margin: $Event-spacing-base
```

Comment:
Again, the component shouldn't set its own margin.
You must think about the component in isolation of everything. That margin only exists because, in the scenario you're using it, when there are Events side by side, you need them to be separated. Well that's not this component's responsibility